reference/1_application_flow/1_3_synchronization/README.md

```diff
@@ -0,0 +1,140 @@
+# Synchronization
+
+Cranq offers the following synchronization primitives to manage application flow:
+
+### Syncer & splitter
+
+The ```flow/Syncer``` aggregates the input signals & forwards it only, once all of it's inputs received a trigger. 
+- It's counterpart is ```flow/Splitter```, which is capable of splitting up the aggregated signal to the original signals.
+- They both have dynamic input/output ports, allowing the developer to scale them as needed

```

Comment:
Maybe use the "spread port" terminology here?
Or should we discuss the official term to use? Dynamic port sounds good as well, can even be easier to understand.

Reply:
Will add both as used later